src/nginx/azext_nginx/aaz/latest/nginx/deployment/_create.py

```diff
@@ -0,0 +1,553 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "nginx deployment create",
+)
+class Create(AAZCommand):
+    """Create an Nginx for Azure resource
+
+    parameters:
+          - name: --name
+             It should contain only alphanumeric characters, up to 30 characters long; and cannot begin or end with a hyphen.
+          - name: --sku
+            The billing information for the resource- https://docs.nginx.com/nginx-for-azure/billing/overview/
+             Usage: --sku name=XXX
+          - name: --network-profile
+            Usage: --network-profile front-end-ip-configuration="<private or public IP address information>" network-interface-configuration="<subnet information>"
+                 front-end-ip-configuration: IP information, public or private IP addresses.
+                 network-interface-configuration: A subnet within your virtual network. This subnet should be delegated to NGINX.NGINXPLUS/nginxDeployments
+
```

Comment:
Parameter help message should be in the argument short-summery or long-summery in aaz-dev-tools.
![image](https://user-images.githubusercontent.com/69238381/191433747-3c5e9d10-4031-4a21-8b91-bbf151316010.png)

Reply:
Done